logging/src/jvmTest/kotlin/io/spine/logging/JvmLoggerSpec.kt

```diff
@@ -89,6 +90,20 @@ internal class JvmLoggerSpec {
         consoleOutput shouldContain "Logging class 1"
         consoleOutput shouldContain "Logging class 2"
     }
+
+    @Test
+    fun `log once per N invocations`() {
```

Comment:
I would also test this feature in a more sophisticated way. I.e. what if we continue invoking?